### DISCUSSION TITLE
Better train depo

### DISCUSSION THREAD

**Discussion by Sliderpro** [voxeltycoon/issues#549] [General]
It would be cool if we could take some carts off the train and store these carts in the depo for later use.

**Comment by katepolak** [comment]
This has been brought up multiple times but I don't see any use for it. You can sell them for the full price so you don't need to store them anywhere.

**Comment by victorleblais** [comment]
The only reason I can see is to avoid inflation by stockpiling vehicles and train cars as soon as possible... But I don't really see why you would do that. :P

@Sliderpro It would be helpful if you gave a more detailed explanation of what you're thinking of, like what it adds to the game that you're missing otherwise, what purposes does it serve, how t would work...

**Comment by RaptorSwire** [comment]
I actually found myself thinking about this as well. Sometimes I wanted to store the current setup for later use. It is not a big deal though....different story is: why are they being 100% fully refunded if they are "used"??
